code/modules/client/preference/preferences_mysql.dm

```diff
@@ -261,21 +260,19 @@
 		player_alt_titles = params2list(query.item[58])
 		organ_data = params2list(query.item[59])
 		rlimb_data = params2list(query.item[60])
-		nanotrasen_relation = query.item[61]
-		speciesprefs = text2num(query.item[62])
+		speciesprefs = text2num(query.item[61])
```

Comment:
I must have missed that bit when i typed the comment initially. Ignore this then.